pkgs/games/openttd/default.nix

```diff
@@ -123,27 +116,22 @@ stdenv.mkDerivation rec {
     "--without-liblzo2"
   ];
 
+  postPatch = ''
+    substituteInPlace src/music/fluidsynth.cpp \
+      --replace "/usr/share/soundfonts/default.sf2" "${soundfont-fluid}/share/soundfonts/FluidR3_GM2-2.sf2"
```

Comment:
```suggestion
      --replace-fail "/usr/share/soundfonts/default.sf2" "${soundfont-fluid}/share/soundfonts/FluidR3_GM2-2.sf2"
```

Reply:
What's the difference here? is --replace just renamed / depreciated?